### ISSUE TITLE
CRM Implementation

### ISSUE THREAD

**Issue by nedventures** [LAMakerspace/ToDo#7]
@myamakerspace to train Ned on basics.

**Comment by nedventures** [comment]
I've scheduled a date and time.

**Comment by myamakerspace** [comment]
- [x] get the last little bit of the volunteer info in there for pete's sake
- [ ] set up email blast component
- [ ] set up donation component
- [ ] set up event component

**Comment by myamakerspace** [comment]
Bugged them again to finish that volunteer data!

**Comment by myamakerspace** [comment]
It's not letting me check a box but, the migration is now complete and I can start your training!  I wish there was a video I could start you off with but, they don't have one yet. However, the interface is very intuitive, it should not take long.

**Comment by nedventures** [comment]
What CRM system are we using?
and all boxes can be officially checked off?

**Comment by myamakerspace** [comment]
No, i still need to get those parts online and learn them myself. But I can teach you basics, and volunteers are all in. 

Commitchange

**Comment by myamakerspace** [comment]
Email Template
1. You might be able to go about this a couple ways:
2. To send an email with a template (and modify the template), select a set of supporters and click "Email". From the dropdown towards the right, you can select a template/draft to load into the email body.
3. To take an existing template, modify it, and save it as another template: first, open up the template; then, make modifications to the body and change the Subject. When you click save, it will be saved as a second draft/template (one thing that may be confusing is that we treat drafts/templates as the same thing)
